src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -94,7 +97,15 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
 
             try
             {
-                exitCode = app.Execute(args);
+                ParseResult result = newCmd.Parse(args);
+                if (result.Errors.Any())
```

Comment:
Could you briefly explain why it's important to call `app.Execute` here?

Reply:
`app.Execute()` runs the command. I am prioritizing running System.CommandLine commands first before falling back on existing commands.

Reply:
If there are errors on parsing migrated System.CommandLine command input, then we fallback to actual Microsoft.CommandLineUtils commands.